src/syscheckd/registry/events.c

```diff
@@ -145,13 +155,13 @@ cJSON *fim_registry_value_json_event(const fim_entry *new_data,
     cJSON_AddStringToObject(data, "value_name", new_data->registry_entry.value->name);
     cJSON_AddNumberToObject(data, "timestamp", new_data->registry_entry.value->last_event);
 
-    cJSON_AddItemToObject(data, "attributes",
-                          fim_registry_value_attributes_json(new_data->registry_entry.value, configuration));
+    //cJSON_AddItemToObject(data, "attributes",
+    //                      fim_registry_value_attributes_json(new_data->registry_entry.value, configuration));
 
     if (old_data != NULL && old_data->registry_entry.value != NULL) {
         cJSON_AddItemToObject(data, "changed_attributes", changed_attributes);
-        cJSON_AddItemToObject(data, "old_attributes",
-                              fim_registry_value_attributes_json(old_data->registry_entry.value, configuration));
+        //cJSON_AddItemToObject(data, "old_attributes",
+        //                      fim_registry_value_attributes_json(old_data->registry_entry.value, configuration));
```

Comment:
Are these statements are necessary? If this are not necessary, you could delete them

Reply:
I left them as placeholders because those are needed event fields that cannot be added currently because dbsync library do not provide them right now.